src/systems/filecoin_markets/storage_market/storage_client.id

```diff
@@ -1,18 +1,62 @@
 import ipld "github.com/filecoin-project/specs/libraries/ipld"
-import deal "github.com/filecoin-project/specs/systems/filecoin_markets/deal"
-import piece "github.com/filecoin-project/specs/systems/filecoin_files/piece"
+import deal "github.com/filecoin-project/specs/systems/filecoin_markets/storage_market/storage_deal"
+import actor "github.com/filecoin-project/specs/systems/filecoin_vm/actor"
+import libp2p "github.com/filecoin-project/specs/libraries/libp2p"
+import addr "github.com/filecoin-project/specs/systems/filecoin_vm/actor/address"
+import block "github.com/filecoin-project/specs/systems/filecoin_blockchain/struct/block"
+
+type ClientDeal struct {
+    ProposalCid     ipld.CID
+    Proposal        deal.StorageDealProposal
+    State           StorageDealStatus
+    Miner           libp2p.PeerID
+    MinerWorker     addr.Address
```

Comment:
Worker seems like the wrong address here - shouldn't it know the actor address (from which it can lookup the worker... and the peerid)

Reply:
In this case it's information tracked about a deal, so it makes better sense to keep it on hand or it will have to keep looking it up over the course of the deal -- see new comment when changes are posted